app/(dashboard)/mailboxes/[mailbox_slug]/saved-replies/savedReplyForm.tsx

```
@@ -16,9 +15,10 @@ import {
   AlertDialogTrigger,
 } from "@/components/ui/alert-dialog";
 import { Button } from "@/components/ui/button";
-import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "@/components/ui/form";
+import { Form, FormControl, FormDescription, FormField, FormItem, FormLabel, FormMessage } from "@/components/ui/form";
```

ESLint

[error] 'FormDescription' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)